clients/go/pkg/zbc/client_test.go

```diff
@@ -164,6 +166,61 @@ func (s *clientTestSuite) TestGatewayAddressEnvVar() {
 	s.EqualValues(fmt.Sprintf("0.0.0.0:%s", parts[len(parts)-1]), config.GatewayAddress)
 }
 
+func (s *clientTestSuite) TestDefaultUserAgent() {
+	// given
+	var incomingContext = make(map[string][]string)
```

Comment:
:question: Found no better way to verify it. Happy to hear your thoughts

Reply:
Other than moving `createServerWithUnaryInterceptor` to some sort of test utility file there's nothing I can think of improving here. We want to test that whoever serves go client requests receives a user-agent header and that that header includes the name and versions so this test looks spot on :+1: